--- wireless_tools.30.orig/Makefile	2025-05-04 00:01:27.320904532 +0900
+++ wireless_tools.30/Makefile	2025-05-04 00:01:47.811414668 +0900
@@ -147,7 +147,7 @@
 
 # Compilation of the dynamic library
 $(DYNAMIC): $(OBJS:.o=.so)
-	$(CC) -shared -o $@ -Wl,-soname,$@ $(STRIPFLAGS) $(LIBS) -lc $^
+	$(CC) -shared -o $@ -Wl,-soname,$@ $(LDFLAGS) $(STRIPFLAGS) $(LIBS) -lc $^
 
 # Compilation of the static library
 $(STATIC): $(OBJS:.o=.so)
